satellite-software/src/LSM9DS1Sensor.cpp

```diff
@@ -349,4 +349,46 @@ void LSM9DS1Sensor::setGysroSensitivitySetting(GyroSensitivitySetting setting) {
 void LSM9DS1Sensor::setMagneticSensitivitySetting(MagneticSensitivitySetting setting) {
     eeprom_update_byte((uint8_t *)(getEEPROMBaseAddress()+OFFSET_MAGNETIC_SENSITIVITY), setting);
     _magneticSensitivity = setting;
+}
+
```

Comment:
This `handleCommand()` method should be braced by `#ifdef ENABLE_AMBASAT_COMMANDS` so that it is not compiled in if someone is building the code for using the internal ATmega328P oscillator (and thus cannot even try to receive downlinks). I realize my original code missed doing that in the BME680 code ... please feel free to fix that too. And don't get get the headers.

Reply:
Done for LSM9DS1 and BME680